scripts/generateManifestForRustAdblock.js

```diff
@@ -34,6 +34,9 @@ const generateManifestFile = async (name, base64PublicKey, uuid) => {
 const generateManifestFileForDefaultAdblock =
   generateManifestFile.bind(null, 'Default', defaultAdblockBase64PublicKey, 'default')  // eslint-disable-line
 
+const generateManifestFileForDefaultPlaintextAdblock =
+  generateManifestFile.bind(null, 'Default (plaintext)', defaultPlaintextPubkey, defaultPlaintextComponentId)  // eslint-disable-line
```

Comment:
This is not really related to this PR, but what's the es-lint rule that fails here? Maybe we should disable it globally instead of ignoring any warnings/errors on all of these lines?

Reply:
Hmm, good call. `npm run lint` actually uses `standard`, there's no `eslint` at all here. I can remove those comments.